samples/secrets-manager/readme.md

```diff
@@ -0,0 +1,3 @@
+# Spring Cloud AWS Secrets Manager Config Sample
+
+This sample application shows how to retrieve a secret from AWS Secrets Manager using Spring Cloud AWS and Spring Boot [Config Import](https://docs.spring.io/spring-boot/docs/2.4.0/reference/html/spring-boot-features.html#boot-features-external-config-files-importing) feature=]]]]]]]]]]]
```

Comment:
Should we provide link to https://cloud.spring.io/spring-cloud-aws/spring-cloud-aws.html#_integrating_your_spring_cloud_application_with_the_aws_secrets_manager as well?